tests/openssl_test.py

```diff
@@ -0,0 +1,15 @@
+from tests import LimitedTestCase, main
+
+class TestOpenSSL(LimitedTestCase):
```

Comment:
I like the TestCase API (defining methods). I used queue_test.py as a template, this file calls main().

Please rewrite the test as you want, I don't care on how tests are written, I only care of having this bug fixed :-)